### ISSUE TITLE
Shield Battery hotkey menu shows 2 Recharge buttons & unset hotkey warning

### ISSUE THREAD

**Issue by Ahli** [Ahli/sc2xml#99]
Shield Battery has two Recharge buttons in the hotkeys menu (but only there)

You can still right click the battery, so this is probably only annoying with its undefined hotkey warning

![image](https://github.com/Ahli/sc2xml/assets/5763784/541c6087-6c96-44a2-928d-28c3166a3782)

Shield Battery's command card is buggy in the engine for some reason. It looks differently in the editor than in game and in the hotkey menu it apparently looks different again...

**Comment by Ahli** [comment]
When you look at it online, the cancel button is missing as well in hotkey menu and during gameplay.
![image](https://github.com/Ahli/sc2xml/assets/5763784/48129233-241d-4376-a96f-e2a233dd1b1c)

Also, the stop button does not exist online.
![image](https://github.com/Ahli/sc2xml/assets/5763784/6324ed7a-5ee8-4295-bbe8-963ea5205d54)

I do not know why editor tests and online gameplay have these different behaviors, but that is really strange.

The only strange thing with Shield Battery is that it uses abilities and effects that use @ character in their IDs... potentially that breaks something...

**Comment by Ahli** [comment]
TLDR:
- has no Cancel button during online play (but in editor tests)
- has no Stop button during online play (but in editor tests)
- has two Recharge buttons in hotkey menu which create a warning about an unset hotkey
- has no Cancel button in hotkey menu (but in editor tests)
